### ISSUE TITLE
Uglify issue with async-computed?

### ISSUE THREAD

**Issue by dsl101** [foxbenjaminfox/vue-async-computed#24]
When I try to build my Vue / Quasar app, I get this error during the build process:

    ERROR in js/vendor.js from UglifyJs
    Unexpected token: punc (() [js/vendor.js:62695,8]

That line is the `get ()` definition in this function:

    function makeLazyComputed (key) {
      return {
        get () {
          this[lazyActivePrefix + key] = true;
          return this[lazyDataPrefix + key]
        },
        set (value) {
          this[lazyDataPrefix + key] = value;
        }
      }
    }

which I'm guessing is coming from the vue-async-computed module. Any idea why Uglify would object here? You'd mentioned before about Babel settings - perhaps something still isn't quite right?

**Comment by Blizzke** [comment]
Have exactly the same issue at this point, it is "more recent" (last 2 months or so) as it has always worked for me so far

**Comment by dsl101** [comment]
I think since the source was split into 2 files (with `lazy.js` being added), the transpilation isn't working right, and so some ES6 is slipping through... Still don't have a fix though - I don't need to build just yet, so am hoping a 'proper' fix will come rather than a workaround (like replace `lazy.js` with a pre-transpiled version)

**Comment by foxbenjaminfox** [comment]
Sorry it took me a week to fix this. I just now started a new job, and I've been rather busy.

By the way, the problem before with Babel was simply that Babel was just concatenating the files, and not properly resolving imports. To solve that, I set up Rollup to bundle the files, and Babel to transpile the bundle.

Then, in addition to having a `main` field in `package.json`, I also followed Rollup's recommendation to add a [`module`](https://github.com/rollup/rollup/wiki/pkg.module) field, in order to allow the package to be imported as an ES6 module.

That field then pointed to the bundled but untranspiled code. That's the problem. Turns out I got it exactly reversed: `module` is supposed to point to code which is [unbundled, but transpiled](](https://github.com/rollup/rollup/wiki/pkg.module#wait-it-just-means-import-and-export--not-other-future-javascript-features)!

Quasar (and other bundling tools) are smart enough to import the `module` if that field exists, and expect the code that's there to be transpiled. In this case, that wasn't true, causing this issue.

I have released version `3.3.1`, which solves this problem by removing the `module` field.
